### PR TITLE
Adding Support for Linux

### PR DESCRIPTION
**Pull request by noelmiller**
anojht, 

I appreciate all the work you did in making tool open source! I've found the closed source version of the tool for Linux is very poorly documented and was a pain in the butt to get working. 

I came across your project and made some edits in my fork of the project that I would love to get into the main project! The main change was changing the logging location. Once I made that change and got all the dependencies installed, the project ran perfectly on Fedora. 

I have also updated the README file to include how to get the project installed on Linux.

I unfortunately do not have a Macbook to test if I broke functionality, so feel free to reply with changes needed to be made in this pull request. 

This is my first pull request on a project, so I'm hoping that this isn't too long or wordy. 

Have a great weekend! 

Noel Miller (Xmas13)
noelmiller@protonmail.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +84,7 @@ If you choose to install dependencies manually as per the instructions above, yo
 
 ## Troubleshooting App Issues
 
-The app is set to create DEBUG logs in the following location: `~/Library/Logs/com.anojht.opensourcesynologycloudsyncdecrypttool.log`
+The app is set to create DEBUG logs in the following location: `~/Library/Logs/com.anojht.opensourcesynologycloudsyncdecrypttool.log or ~/com.anojht.opensourcesynologycloudsyncdecyrpttool.log in the case of Linux.`
```
Comment (anojht) [inline README.md]: Here the name of the log file should be changed to reflect the changes you made to the name of the file in the code.

**Review by anojht — COMMENTED** [review]
Please see the review comments and address them. I notice that a swap file has been committed in your pull request. Please remove the swap file as it is a temporary file generated by vim when editing.

**Comment by anojht** [comment]
@Xmas13 Thanks for your kind words and contribution to the project, it is greatly appreciated. Sorry for getting to it so late as I have been busy with work. For the most part the changes are non breaking and they are good, can you please review and address the comments I have left in your commits.

**Comment by noelmiller** [comment]
@anojht 
So sorry for the slow response to this. I will do what I can to make the changes you requested and then we can merge. It has been a busy holiday season and life is just busy in general!

**Comment by anojht** [comment]
Thanks for your commit, I am merging these changes.

**Pull request merged**